solution/backend/resources/admin.py

```diff
@@ -285,7 +285,7 @@ def format_value(self, value):
                 removals = self.locations_to_strings(row["removals"])
                 bulk_adds = self.locations_to_strings(row["bulk_adds"])
                 date = parse_datetime(row["date"]).strftime("%Y-%m-%d at %I:%M %p")
-                output.append(f"{i+1}: On {date}, {row['user']} %s%s%s%s%s." % (
+                output.append(f"{i + 1}: On {date}, {row['user']} %s%s%s%s%s." % (
```

Comment:
the Ruff github action began throwing `solution/backend/resources/admin.py:288:35: E226 Missing whitespace around arithmetic operator` out of nowhere; this fixes the issue